apps/src/templates/certificates/Certificate.jsx

```diff
@@ -131,53 +138,73 @@ function Certificate(props) {
+          {i18n.congratsCertificateHeading()}
+        </Heading1>
+      </div>
       {tutorial && (
         <LargeChevronLink
           link={`/s/${tutorial}`}
           linkText={i18n.backToActivity()}
         />
       )}
-      <div id="uitest-certificate" style={certificateStyle}>
-        <BackToFrontConfetti active={personalized} style={styles.confetti} />
-        <a href={certificateShareLink}>
-          <img src={imgSrc} />
-        </a>
-      </div>
-      <div style={certificateStyle}>
-        {tutorial && !personalized && (
-          <div>
-            <h2>{i18n.congratsCertificatePersonalize()}</h2>
-            <input
-              id="name"
-              type="text"
-              style={styles.nameInput}
-              placeholder={i18n.yourName()}
-              ref={nameInputRef}
-            />
-            <button
-              type="button"
-              style={styles.submit}
-              onClick={personalizeCertificate.bind(this, certificateId)}
-            >
-              {i18n.submit()}
-            </button>
-          </div>
-        )}
-        {tutorial && personalized && (
-          <div>
-            <h2 id="uitest-thanks">{i18n.congratsCertificateThanks()}</h2>
-            <p>{i18n.congratsCertificateContinue()}</p>
-          </div>
-        )}
-        <h2>{i18n.congratsCertificateShare()}</h2>
-        <SocialShare
-          facebook={facebook}
-          twitter={twitter}
-          print={print}
-          under13={under13}
-        />
+      <div className={style.certificateContainer}>
+        <div id="uitest-certificate" className={certificateStyle}>
+          <BackToFrontConfetti
+            active={personalized}
+            className={style.confetti}
+          />
+          <a href={certificateShareLink}>
+            <img src={imgSrc} />
+          </a>
+        </div>
+        <div className={`${certificateStyle} ${style.inputContainer}`}>
+          {tutorial && !personalized && (
+            <div>
+              <Heading3>{i18n.congratsCertificatePersonalize()}</Heading3>
+              <BodyThreeText className={style.enterName}>
+                {i18n.enterYourName()}
+              </BodyThreeText>
+              <div className={style.inputButtonContainer}>
+                <input
+                  id="name"
+                  type="text"
+                  className={style.nameInput}
+                  placeholder={i18n.yourName()}
+                  ref={nameInputRef}
+                />
+                <button
+                  type="button"
+                  className={style.submit}
+                  onClick={personalizeCertificate.bind(this, certificateId)}
+                >
+                  {i18n.submit()}
+                </button>
+              </div>
+            </div>
+          )}
+          {tutorial && personalized && (
+            <div>
+              <Heading2>
+                <div id="uitest-thanks">{i18n.congratsCertificateThanks()}</div>
```

Comment:
Is this id made for ui tests being used anywhere?

Reply:
Yes its being used in `hour_of_code_finish.feature`. The drone build was failing as the Heading component doesnt allow for ids as a prop.